src/DiagramGenerator/Diagram.php

```diff
@@ -154,7 +155,10 @@ public function draw()
 
         $this->image->setImageFormat($this->board->getImageFormat());
         if ($this->image->getImageFormat() === Texture::IMAGE_FORMAT_JPG) {
-            $this->image->setImageCompressionQuality(self::COMPRESSION_QUALITY_JPG);
+            $compressionQualityJpg = is_null($this->config->getCompressionQualityJpg()) ?
```

Comment:
Might be nicer:
`$compressionQualityJpg = $this->config->getCompressionQualityJpg() ?: self::COMPRESSION_QUALITY_DEFAULT_JPG`